_data/simple-icons.json

```
@@ -15321,6 +15321,11 @@
             "hex": "1343FB",
             "source": "https://www.telequebec.tv/societe/logo-et-normes-techniques"
         },
+        {
+            "title": "Telefonica",
```

Here you don't need to encode it:

```suggestion
            "title": "Telefónica",
```